docs/google/engine.md

```diff
@@ -129,6 +129,14 @@ has a method `run_batch()` that functions similar to `run()` but accepts a
 list of circuits and parameter sweeps.  Each circuit must have a corresponding
 parameter sweep.  If the circuit does not use a sweep, pass in `None`.
 
+There are some restrictions on the circuits that can be batched together.
+All circuits in the same batch must measure the same set of qubits.  They
```

Comment:
Use bullet points and spell out the rules, makes it more memorable: e.g.: 

* **Same qubits**: All circuits in the same batch must measure the same set of qubits.
* **Same repetitions**: All circuits in the same batch must have the same number of repetitions.

Reply:
Good idea.  Done.